src/main/org/deidentifier/arx/criteria/EDDifferentialPrivacy.java

```diff
@@ -98,9 +91,9 @@ public EDDifferentialPrivacy(double epsilon, double delta,
         this.epsilon = epsilon;

```

Comment:
Check comment "Note: ...". Also check, if changing the constructor breaks anything.